code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/NetworkInfoDaoImpl.java

```diff
@@ -340,6 +341,10 @@ protected HostPortForwardData map(List<Object> input) {
                 .on(port.PRIVATE_IP_ADDRESS_ID.eq(privateIpAddress.ID))
             .join(publicIpAddress)
                 .on(port.PUBLIC_IP_ADDRESS_ID.eq(publicIpAddress.ID))
+            .join(NIC)
+                .on(INSTANCE_HOST_MAP.INSTANCE_ID.eq(NIC.INSTANCE_ID))
+            .join(NETWORK_SERVICE)
```

Comment:
Since your joining some new tables I'm guess some more REMOVED.isNull() need to be added